integreat_cms/api/v3/pages.py

```diff
@@ -142,7 +143,7 @@ def get_single_page(request, language_slug):
         # Strip leading and trailing slashes to avoid ambiguous urls
         url = request.GET.get("url").strip("/")
         # The last path component of the url is the page translation slug
-        page_translation_slug = url.split("/")[-1]
+        page_translation_slug = slugify(url.split("/")[-1])
```

Comment:
```suggestion
        page_translation_slug = slugify(url.split("/")[-1], allow_unicode=True)
```
I guess we have to allow unicode slugs here. Timing should be irrelevant in comparison to the database queries etc, right? Otherwise we could think about using the slugify method only if the regular url did not return a response...